### ISSUE TITLE
(BT) crash when loading callvoted map

### ISSUE THREAD

**Issue by bulldozerecske** [openmoh/openmohaa#476]
I was playing on mp_brest_dm then I initiated a callvote to change map to mp_berlin_tow.

Dedicated server console shows:

```
♥Bulldozer-HUN-OpenMohaa called a vote.
♥Vote Passed: Tug-of-War Map Berlin - mp_berlin_tow
------- S_StopAllSounds (don't stop music) -------
------- S_StopAllSounds Complete-------
Called FadeSound with: 0.000000
Called FadeSound with: 0.000000
------ Unloading ClientGame ------
------- S_StopAllSounds (stop music) -------
MUSIC: changing from [ normal | normal ] to [ none | none ]
------- S_StopAllSounds Complete-------
Called FadeSound with: 0.000000
------ Unloading ClientGame ------
------- S_StopAllSounds (stop music) -------
------- S_StopAllSounds Complete-------
Called FadeSound with: 0.000000
WARNING: server is not allowed to set pmove_msec=0
WARNING: server is not allowed to set pmove_fixed=0
Called FadeSound with: 0.000000
Setting up Shaders
------- Sound Begin Registration -------
------- S_StopAllSounds (stop music) -------
------- S_StopAllSounds Complete-------
------- Sound Begin Registration Complete -------
------ Unloading ClientGame ------
Trying to load "cgame.x86_64.dll" from "C:\Program Files (x86)\GOG.com\Medal of Honor"...
Sys_GetCGameAPI(cgame.x86_64.dll): succeeded ...

------------------
Event system initialized: 8 classes 162 events 10368 total memory in response list

CM_LoadMap( maps/obj/MP_Berlin_TOW.bsp, 1 )
UI_DrawConnect called


-----------PARSING UBERSOUND------------
Any SetCurrentTiki errors means that tiki wasn't prefetched and tiki-specific sounds for it won't work. To fix prefetch the tiki. Ignore if you don't use that tiki on this level.
CG_Command_ProcessFile: ubersound/ubersound.scr
Parse/Load time: 0.049000 seconds.
-------------UBERSOUND DONE---------------



-----------PARSING UBERDIALOG------------
Any SetCurrentTiki errors means that tiki wasn't prefetched and tiki-specific sounds for it won't work. To fix prefetch the tiki. Ignore if you don't use that tiki on this level.
CG_Command_ProcessFile: ubersound/uberdialog.scr
Parse/Load time: 0.157000 seconds.
-------------UBERDIALOG DONE---------------

WARNING: shader 'textures/general_structure/window4_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window725_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window725_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window725_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window725_frame' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window725_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window725_frame' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window725_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window725_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_broken' has depthmask enabled after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has opaque maps defined after stage 0!!!
WARNING: shader 'textures/general_structure/window4_frame' has depthmask enabled after stage 0!!!
stitched 2 LoD cracks
...loaded 17001 faces, 311 meshes, 0 trisurfs, 0 flares
Cmd_AddCommand: ter_restart already defined
WARNING- DOWNGRADING TO OLD ANIMATION FORMAT FOR FILE: models/static/german_rations_1/german_rations_1.skc
```

Client crashed:

![image](https://github.com/user-attachments/assets/93c08d34-9f6e-4640-8385-e8092730b864)

Version: OpenMoHAA 0.71.0-unstable+195.edb52a8 win_msvc64-x86_64 Oct 23 2024 UTC

**Comment by smallmodel** [comment]
Can't reproduce as usual, works well on my machine. No idea about what mods are installed, what was the gametype before callvote etc

**Comment by bulldozerecske** [comment]
> Can't reproduce. No idea about what mods are installed, if the server is Vanilla or openmohaa, what was the gametype before callvote etc

No mods and this is the server.cfg. Server is OpenMoHAA 0.71.0-unstable+195.edb52a8 win_msvc64-x86_64 Oct 23 2024 UTC

```
// Server Name
// Name that will appear on the server browser.
sv_hostname "Bulldozer's OpenMoHAA test server"

// Game Type.
// 1 Free for All
// 2 Team Based
// 3 Round Based
// 4 Objective
// 5    Of War
// 6 Liberation
set g_gametype 2

// countdown
// Settings for Countdown mod.
//set countdown 1
//set countdown_death_print 1
//set countdown_time 10:00
//set countdown_rounds 0
//set countdown_light 1
//set countdown_hold_for 0
//set countdown_locate 5
//set countdown_idle 100

// Clock time
// Clock appear in the map.
set clock 1

// Map fix
//set mapfix 1

// Team balance
// Balance the teams in the game.
set g_teambalance 0

// Health Drop
// When enabled players will drop a health pack when killed.
set g_healthdrop 1

// Realism
// Toggles reaslistic mode.
set g_realismmode 0

// Fast Run Speed
// When checked, the speed at which players can run through levels is increased. 
// Leaving it unchecked defaults to Medal of Honor Allied Assault's original run speed.
//set sv_dmspeedmult 1.100000

// Team Damage
// Toggles friendly fire.
set g_teamdamage 0

// Heal Rate
// Determines how fast players heal when they use a health pickup.
set g_healrate 25

// Max Players
set sv_maxclients 12

// Frag Limit
// The score at which the player or team will win the current map.
fraglimit 000

// Time Limit
// This sets the elapsed time at which the game ends, the winner being the player with the highest score at that time.
timelimit 0

// Round Reset Time
// This sets how long each round will take, in minutes. This can be smaller than the map rotation time, but not larger.
//set roundlimit 0

// Map Rotation Time
// This sets the maximum amount of time in minutes players will spend in a particular map.
//set maprotationtime 0

// Team Spawn Delay
// This sets a delay in seconds between spawns. This is useful if you would like the 
// player spawned in batches, rather than one at a time.
set sv_team_spawn_interval 0

// Inactive Spectate
// This sets the time in seconds after which an inactive player becomes a spectator. To save server bandwidth, 60 seconds is default.
set g_inactivespectate 200

// Inactive Kick
// This sets the time in seconds after which an inactive player is kicked out of the game. 900 (15 minutes) seconds is default.
set g_inactivekick 0

// Use Gamespy
// To use GameSpy gaming service for internet-based multiplayer games.
//set sv_gamespy 1
//set sv_master1 "mohmaster.2015.com"
//set sv_master2 "master0.gamespy.com"
//set sv_master3 "master1.gamespy.com"
//set sv_master4 "clanservers.net"

// Pure
// Verifies that the files on the client are the same as those on the server.
set sv_pure 0

// Allow Vote
// Enables players to call a vote.
set g_allowvote 1

// Force Team Spectate
set dmflags 67371016
set g_forceready 0
set g_forcespawn 0

// Invulnerable Time
// Amount of time a player is invulnerable for after spawning.
set sv_invulnerabletime 3

// Team Kill Warning
// Amount of team kills before the player is warned.
set g_teamkillwarn 2

// Team Kill Kick
// Amount of team kills before the player is kicked off the server.
set g_teamkillkick 3

// Team Switch Delay
// Frequency at which you can switch teams.
set g_teamswitchdelay 30


// Allow Join Time
set g_allowjointime 30

// Flood Protect
// Limits amount of text a player can type.
set sv_floodprotect 1

// Forces players to only be able to spectate behind their own team. 
//set g_forceteamspectate 0

// Invulnerable Time
// Amount of time (in seconds) a player is invulnerable for after spawning (default=3 seconds)
//set sv_invulnerabletime 3

// Private Clients
// Clients that require a password to join.
set sv_privateclients 0

// Server Password
// Allows you to set a password for the server.
set sv_privatepassword ""

// Keywords
// Keywords which allow browsers to filter servers.
set sv_keywords ""

// Net Port
set net_port 12203

// precache
set sv_precache 1

// GS-Pro
set chater 0
set developer 3
set logfile 3
set g_logSync 0

// Console
set dedicated 1
set ui_console 1
//set g_logstats 1
//set g_log "qconsole.log"
//set g_logSync 1
//set developer 2 //2
//set logfile 2 //console log - 0=no log 1=buffered 2=continuous 3=append 
//set chat 1
//set sv_chatter 1

// Cheats
set cheats 1

//Open or close download.
set sv_allowdownload 0

// Default to no-sprint and original Allied Assault runspeed.
//sv_sprinton 0

// Run Speeed
//set sv_runspeed 293	// spearhead runspeed is 287

// Minimum Ping
// Minimum Ping to allow clients to join with. 0 means anyone.
set sv_minping 0

// Maximum Ping
// Maximum Ping to allow clients to join with. 0 means anyone.
set sv_maxping 0


// Maximum Rate
set sv_maxrate 0

// Fix lag
set sv_maxpackets 15

// Map
// Starting map on the rotation.
//map dm/mohdm1

// Map Rotation List
//sv_maplist "dm/mohdm1 dm/mohdm2 dm/mohdm3 dm/mohdm4 dm/mohdm6 dm/mohdm7 dm/mp_bahnhof_dm dm/mp_bazaar_dm dm/mp_brest_dm dm/mp_stadt_dm dm/mp_unterseite_dm dm/mp_verschneit_dm"

set x1 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/MP_Unterseite_DM; set nextmap vstr x2"
set x2 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm2; set nextmap vstr x3"
set x3 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm3; set nextmap vstr x4"
set x4 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/MP_Brest_DM; set nextmap vstr x5"
set x5 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm4; set nextmap vstr x6"
set x6 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/MP_Bazaar_DM; set nextmap vstr x7"
set x7 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm5; set nextmap vstr x8"
set x8 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm6; set nextmap vstr x9"
set x9 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm7; set nextmap vstr x10"
set x10 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/MP_Stadt_DM; set nextmap vstr x11"
set x11 "set g_gametype 1; maprotationtime 60; fraglimit 0; timelimit 20; roundlimit 0; map dm/mohdm1; set nextmap vstr x1"

seta sv_maplist ""

vstr x1 //startup first map

// Rconpassword
set rconpassword "test"
```

**Comment by smallmodel** [comment]
Fixed it, in commit 54b9a0c340c5e62a12ab251f6325f0d751403fe8
